app/app/components/MemorableLines.module.css

```diff
@@ -0,0 +1,53 @@
+.memorableLines {
+    width: 100%;
+    padding: 40px 20px;
+    text-align: center;
+    margin: 50px 0 0 0;
+}
+
+.linesContainer {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.line {
+    max-width: 500px;
+    margin: 20px;
```

Comment:
I can see the `20px` recurring quite often, it might be a good idea to register a CSS variable in the `globals.css`.
See how it was implemented in the `React.js 1` Homework.